docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
 Column-based Noise
 ==================
 
+Column-based noise introduces errors in individual fields within a row of data.
+The types implemented in pseudopeople are listed below, along with instructions
+about how to configure them.
+
+.. contents::
+   :depth: 2
+   :local:
+
 Leave a field blank
 -------------------
 
+When this noise type occurs, whatever value was supposed to be collected is missing for this row.
+This could be because the input was left blank, an answer was refused,
+or the answer was illegible or unintelligible.
+pseudopeople will output a :code:`numpy.nan` value in
+the relevant cell.
+
+This noise type is called :code:`leave_blank` in the configuration. It takes one parameter:
+
+.. list-table:: Parameters to the :code:`leave_blank` noise type
```

Comment:
I don't like how the code font looks in the table title -- maybe just write leave_blank in normal text here.